src/wp-includes/js/media/views/attachment/details.js

```diff
@@ -2,22 +2,18 @@ var Attachment = wp.media.view.Attachment,
 	l10n = wp.media.view.l10n,
 	Details;
 
-/**
- * wp.media.view.Attachment.Details
- *
- * @memberOf wp.media.view.Attachment
- *
- * @class
- * @augments wp.media.view.Attachment
- * @augments wp.media.View
- * @augments wp.Backbone.View
- * @augments Backbone.View
- */
 Details = Attachment.extend(/** @lends wp.media.view.Attachment.Details.prototype */{
 	tagName:   'div',
 	className: 'attachment-details',
 	template:  wp.template('attachment-details'),
 
+	/**
+	 * Overrides the attributes method in the Attachment prototype.
```

Comment:
Only describes that attributes is overridden, not what the new attributes method is actually doing.